packages/git-raw-commits/test.js

```diff
@@ -29,8 +29,6 @@ describe('git-raw-commits', function () {
       })
       .pipe(through(function () {
         done('should error')
-      }, function () {
-        done('should error')
```

Comment:
The flush function _does_ get called before `error` is emitted now, so this is removed to not call `done` twice.

That seems okay... I couldn't think of a reason that should be a problem, and all tests are still passing.